test/MUXControlsTestApp/VisualTreeTestBase.cs

```diff
@@ -0,0 +1,321 @@
+﻿using Common;
+using System;
+using System.Threading.Tasks;
+using Windows.Storage;
+using System.IO;
+using MUXControls.TestAppUtils;
+using MUXControlsTestApp.Utilities;
+using System.Text;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Markup;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using PlatformConfiguration = Common.PlatformConfiguration;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    public enum Theme
+    {
+        Dark,
+        Light,
+        HC,
```

Comment:
HC should be 'HighContrast' but that's a separate flag, eh?  As you can have HC+Dark and HC+Light.  Doesn't look like any High Contrast scenarios are covered yet?

Reply:
I don't know how to do with HC yet, here is just a placeholder and I will remove HC.

---
In reply to: [245381662](https://github.com/Microsoft/microsoft-ui-xaml/pull/126#discussion_r245381662) [](ancestors = 245381662)